### DISCUSSION TITLE
Modelleerprincipes - Definitie gegeven

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#49] [General] [answered]
[Definitie van gegeven](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#dfn-gegevens) luidt nu:

> Een gegeven is een kennisuitdrukking over een onderwerp. Een gegeven bestaat uit een verwzijzing naar een object (het onderwerp), een kenmerk / eigenschap / predicaat en een waarde.

Tijdens de [laatste expertgroep](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/27-05-2021-aantekeningen.md) meeting hadden we hier discussie over:

> "Lennart ziet een linked data bias. Michel ziet het als een generieke beschrijving (het gaat om key-value pairs), wat pas invulling in linked data zou krijgen wanneer je het ook over subjects/objects hebt."

**Comment by pmaria** [comment]
Voorstel van @lennartvanbergen-kadaster voor een minder biased formulering: 
> Een gegeven is de waarde, het feit, die we hebben waargenomen of geconstateerd voor een eigenschap van een object.

**Reply by pmaria** [reply]
"Feit" vind ik te sterk geformuleerd. Volgens mij kunnen we dit ook prima achterwege laten.

"Waargenomen of geconstateerd" vind ik iets te restrictief geformuleerd.  We zouden dit kunnen veralgemeniseren naar 'uitgedrukt'.

Voorstel:

**"Een gegeven is de waarde die is uitgedrukt voor een eigenschap van een object".**

Wat vinden jullie hiervan @lennartvanbergen-kadaster @MichelBohms en anderen?

**Comment by lennartvanbergen** [comment]
_Gegevens zijn de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden._ 

_Een feit is een gebeurtenis of omstandigheid waarvan de werkelijkheid vaststaat, doordat het ofwel zintuiglijk waargenomen ofwel instrumenteel gemeten kan worden. Een feit moet met objectieve waarnemingen vastgesteld en getoetst kunnen worden (zie het artikel wetenschappelijke methode)._

Ad. gemeten: dat is ook een vorm van waarnemen. 

Wellicht wat filosofisch maar als ik naar een object in de werkelijkheid kijk (of luister), dan neem ik daar een eigenschap van waar. Ik win wat gegevens in over dat object. Soms neem ik dat direct waar, soms constateer ik het een aantal maanden later en weer ik niet precies vanaf wanneer die gegevens zoals nu waargenomen gelden waren voor dat object - en dat laatste bedoeld ik met geconstateerd. 

'uitgedrukt is' voelt voor mij meer als: er staat data in de database, het veld heet X, de waarde van X is die data, zo hebben we het uitgedrukt. 

Voor mij was/is de definitie zoals ik 'm typte, goed. Dan blijven we ook veel dichter bij de beschrijving van een gegeven in MIM.

**Reply by pmaria** [reply]
> Gegevens zijn de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden.
>
> Een feit is een gebeurtenis of omstandigheid waarvan de werkelijkheid vaststaat, doordat het ofwel zintuiglijk waargenomen ofwel instrumenteel gemeten kan worden. Een feit moet met objectieve waarnemingen vastgesteld en getoetst kunnen worden (zie het artikel wetenschappelijke methode).
>
> Ad. gemeten: dat is ook een vorm van waarnemen.
>
> Wellicht wat filosofisch maar als ik naar een object in de werkelijkheid kijk (of luister), dan neem ik daar een eigenschap van waar. Ik win wat gegevens in over dat object. Soms neem ik dat direct waar, soms constateer ik het een aantal maanden later en weer ik niet precies vanaf wanneer die gegevens zoals nu waargenomen gelden waren voor dat object - en dat laatste bedoeld ik met geconstateerd.

Voor objecten in de werkelijkheid volg ik je, maar je kunt ook gegevens hebben over objecten die je niet kunt waarnemen in de werkelijkheid. Bijvoorbeeld virtuele objecten. Daar past deze definitie zie zo goed.

> 'uitgedrukt is' voelt voor mij meer als: er staat data in de database, het veld heet X, de waarde van X is die data, zo hebben we het uitgedrukt.

Met uitgedrukt bedoelde ik "een gedachte in taal verwoorden". Een uitdrukking. Een utterance of statement in het Engels.

Een gegeven hoeft geen feit te zijn. En een gegeven hoeft geen waarneming of constatering te zijn mijns inziens. Vandaar dat ik op zoek ben naar een algemenere formulering die voor alle soorten objecten de lading dekt.

**Comment by MichelBohms** [comment]
In LD denk ik aan een Statement: oftwel een combinatie van waarde/ref ("object") voor een eigenschap/relatie van/voor een subject (instantie of klasse etc.).
Dus wat krijg je als je dit generaliseert (of beter: LD-neutraliseert) ?......
in natuurlijke taal is het een "onderwerp gezegde lijdend voorwerp"

feit is een moeilijk term....soms ziet men triple als "feit" (synoniem van "statement") soms is het meer richting een meetwaarde.

in het Engels heb je onderscheid beter: measure versus measurement...kwam ik tegenaan in w3c-ssn/sosa: Measure is niet alleen maar bedoeld als meetwaarde!

"statement" is best mooie term eigenlijk (misschien beter dan "feit").

**Reply by pmaria** [reply]
Ja uitdrukking is voor mij ook synoniem met "statement"

**Comment by lvdbrink** [comment]
Ik dacht nog aan "vastgesteld". Maar dat is misschien ook te sterk; bij waarnemingen / metingen is de waarde vaak niet vastgesteld maar geschat.

**Comment by lvdbrink** [comment]
Uit expertgroep meeting 17-6: 

"Ingewonnen"? te procesgericht. "Vastgesteld"? misschien te sterk, maar heeft lichte voorkeur. 

Dit issue is voor niemand blokkerend, maar we moeten wat beter naar alternatieven kijken. Misschien helpt het om dit te vragen aan de Inhoud mensen.

**Comment by lvdbrink** [comment, marked as answer]
Marcel Rietdijk van de Werkgroep Inhoud suggereert: "Opgenomen". Dit vinden Pano en ik allebei een goede suggestie.

**Reply by pmaria** [reply]
@lennartvanbergen-kadaster , @MichelBohms wat vinden jullie van "opgenomen"? Dan zou het worden: 

> "Een gegeven is de waarde die is opgenomen voor een eigenschap van een object".

**Comment by MichelBohms** [comment]
Prima
Klinkt neutraal, los van waarom, bron etc.



Dr. ir. H.M. (Michel) Bohms
Scientist Specialist
Structural Reliability

T +31 (0)88 866 31 07
M +31 (0)63 038 12 20
E ***@***.******@***.***>

Location<http://www.tno.nl/locations/DTS>



***@***.***D76F38.842025B0]<http://www.tno.nl/>

This message may contain information that is not intended for you. If you are not the addressee or if this message was sent to you by mistake, you are requested to inform the sender and delete the message. TNO accepts no liability for the content of this e-mail, for the manner in which you use it and for damage of any kind resulting from the risks inherent to the electronic transmission of messages.




Van: Pano Maria ***@***.***>
Verzonden: vrijdag 2 juli 2021 11:50
Aan: Geonovum/disgeo-imsor ***@***.***>
CC: Bohms, H.M. (Michel) ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [Geonovum/disgeo-imsor] Modelleerprincipes - Definitie gegeven (#49)


@lennartvanbergen-kadaster<https://github.com/lennartvanbergen-kadaster> , @MichelBohms<https://github.com/MichelBohms> wat vinden jullie van "opgenomen"? Dan zou het worden:

"Een gegeven is de waarde die is opgenomen voor een eigenschap van een object".
